WordPress/Classes/Services/BlogService.h

```diff
@@ -58,15 +58,19 @@
 
 - (void)migrateJetpackBlogsToXMLRPCWithCompletion:(void (^)())success;
 
-/*! Syncs an entire blog include posts, pages, comments, options, post formats and categories.
- *  Used for instances where the entire blog should be refreshed or initially downloaded.
- *
- *  \param success Completion block called if the operation was a success
- *  \param failure Completion block called if the operation was a failure
+/**
+ Syncs an entire blog in the background including posts, pages, comments, and blog meta such as
+ post formats, blog options, and categories. Also checks if the blog is multi-author.
+ Used for instances where the entire blog should be refreshed or initially downloaded.
+ */
+- (void)backgroundSyncBlog:(Blog *)blog;
```

Comment:
What if we just stopped doing this? I think we want to keep options synced, and we want formats/categories to be there when editing a post. Everything else we could just load on demand.

Also, why the name change? I'm fine with it, but just curious.